content/fail-learn-reset-transform.md

```diff
@@ -0,0 +1,66 @@
+Title: Fail, Learn, Reset, Transform!
+Date: 2023-03-31
+Category: Posts 
+Tags: engineering, learning, innovation
+Slug: fail-learn-reset-transform
+Author: Willy-Peter Schaub
+Summary: “Success is not final, failure is not fatal, it is the courage to continue that counts.” – Winston Churchill
+
+In [Fail, Learn, Reset](/fail-learn-reset.html) I explored how I encourage everyone in my team and our group to accept failure as an opportunity to learn and how we are trying hard to eradicate the **fear of failure** that often accompanies the **imposter syndrome**. In conversations, I realized that the [Fail, Learn, Reset](/fail-learn-reset.html) post was very engineering focused, but that the concept of learning from failures applies to everything.
+
+>
+> **FAILURE** comes in many shapes and sizes. Here are a few of many definitions from the Merriam-Webster Dictionary:
+>
+> - lack of success
+> - a falling short
+> - a fracturing or giving way under stress
+>
+
+Personally, I prefer of **FAILURE** as an "unexpected outcome" of doing, trying, evolving, or experimenting.
```

Comment:
suggest:
I prefer of **FAILURE**
to:
I prefer to think of **FAILURE**

Reply:
Great feedback.